src/widget/wtracktableview.cpp

```diff
@@ -1389,7 +1389,12 @@ void WTrackTableView::slotReloadTrackMetadata() {
     for (const QModelIndex& index : indices) {
         TrackPointer pTrack = trackModel->getTrack(index);
         if (pTrack) {
-            SoundSourceProxy(pTrack).loadTrackMetadata(true);
+            // The user has explicitly requested to reload metadata from the file
+            // to override the information within Mixxx!
+            SoundSourceProxy(pTrack,
+                    SoundSourceProxy::PARSE_METADATA |
+                    SoundSourceProxy::RELOAD_METADATA_EVEN_IF_ALREADY_PARSED |
+                    SoundSourceProxy::RELOAD_METADATA_EVEN_IF_DIRTY);
```

Comment:
Both later flags are always set as pair, is there a future use case for only one of them? 
If not, we can use a single "force" flag or PARSE_ENSURE_METADATA / PARSE_RELOAD_METADATA